### DISCUSSION TITLE
Is there any sample working app with instructions for how to extend dashboard with App?

### DISCUSSION THREAD

**Discussion by manojLondhe** [saleor/saleor#9702] [Q&A] [answered]
Hi,

With reference to https://docs.saleor.io/docs/3.x/developer/extending/apps/extending-dashboard-with-apps is there any sample working app with instructions for how to extend dashboard with App?

Thanks!

**Comment by karolkielecki** [comment, marked as answer]
Hi @manojLondhe, we're working on some starter packs for apps. However, I'm not sure when they will be available.

**Reply by manojLondhe** [reply]
OK, thank you for quick response
